tests/test_server.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import os
+import logging
+import importlib
+
+
+def test_debug_env_variable_1():
+    os.environ['HPOLIB_DEBUG'] = 'false'
+    from hpolib.container.client_abstract_benchmark import log_level
+    assert log_level == logging.INFO
+
+    os.environ['HPOLIB_DEBUG'] = 'true'
+    import hpolib.container.client_abstract_benchmark as client
+    importlib.reload(client)
+    from hpolib.container.client_abstract_benchmark import log_level
+    assert log_level == logging.DEBUG
+
+
+def test_debug_container():
+    os.environ['HPOLIB_DEBUG'] = 'true'
+    import hpolib.container.client_abstract_benchmark as client
+    importlib.reload(client)
+    importlib.reload(logging)
+    from hpolib.container.benchmarks.ml.xgboost_benchmark import XGBoostBenchmark as Benchmark
+
+    from hpolib.util.openml_data_manager import get_openmlcc18_taskids
+    task_id = get_openmlcc18_taskids()[0]
+
+    b = Benchmark(task_id=task_id,
+                  container_name='xgboost_benchmark',
+                  # container_source='library://phmueller/automl')
+                  container_source='/home/philipp/Dokumente/Code/TabularBenchmarks/Container')
```

Comment:
what's this?

Reply:
I've tried to write a first "whitebox" test for the debugging option. 
It should not crash and show some debug output of the container. 

The reason behind the "importlib.reload" is that I'm using env variables that are read during the import of the file. 
Generally, I think that checking the debug environment variable only on the startup of the hpolib is sufficient because we don't switch from "info" to "debug" and vice versa between runs. 

Is this okay?

Reply:
All fine, I meant the local path.